### PR TITLE
Adding documentation for tests

### PR DESCRIPTION
**Pull request by ashetty1**
As part of issue #781

### CODE REVIEW

**Comment by surajnarwade** [comment]
we have to do `make bin` first right ?

File: script/test_in_openshift/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Functional tests for Kompose on OpenShift
+
+## Introduction
+
+The functional tests for Kompose on OpenShift cluster leverages  oc cluster up` to bring a single-cluster OpenShift instance. The test scripts
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: you forgot to add and extra `

it's suppose to be `oc cluster up`

File: script/test_in_openshift/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+are hosted under script/test_in_openshift. The directory structure is as below:
+
+        script/test_in_openshift/
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: put this structure within a ```

File: script/test_in_openshift/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  
+- [lib.sh](/script/test_in_openshift/lib.sh) consists of helper functions
+  for `kompose up` and `kompose down` checks
+  
+- [tests/](/script/test_in_openshift/tests) directory contains the test scripts
+
+- [compose-files/](/script/test_in_openshift/compose-files/) directory contains the docker compose
+  file used by the test	scripts
+
+- The scripts use [`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) for setting up a single-machine OpenShift cluster. It exits if oc binaries are not installed
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: add `oc`

File: script/test_in_openshift/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- Most of the docker compose files used are the ones already available under [examples/](/examples] or [/script/test/fixtures](/script/test/fixtures).
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: don't add 'most of the docker compose files'

just say `Docker Compose examples are available under examples/`

File: script/test_in_openshift/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Running OpenShift tests
+
+### Install `oc	cluster	up`
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: random spacing between oc and cluster

File: script/test_in_openshift/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+- [compose-files/](/script/test_in_openshift/compose-files/) directory contains the docker compose
+  file used by the test scripts
+
+- The scripts use [`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) for setting up a single-machine OpenShift cluster. It exits if oc binaries are not installed
+
+- Docker Compose examples are available under [examples/](/examples] or [/script/test/fixtures](/script/test/fixtures).
+
+## Running OpenShift tests
+
+### Install `oc	cluster	up`
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: Random spacing?

Install what though?

I think you mean Deploy `oc cluster up`

File: script/test_in_openshift/README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+The scripts use [`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) for setting up a single-machine OpenShift cluster. Installing oc binary is a pre-requisite for running these tests
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: missing period at end of sentence.

File: script/test_in_openshift/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+  
+- [lib.sh](/script/test_in_openshift/lib.sh) consists of helper functions
+  for `kompose up` and `kompose down` checks
+  
+- [tests/](/script/test_in_openshift/tests) directory contains the test scripts
+
+- [compose-files/](/script/test_in_openshift/compose-files/) directory contains the docker compose
+  file used by the test scripts
+
+- The scripts use [`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) for setting up a single-machine OpenShift cluster. It exits if oc binaries are not installed
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: missing period at end of sentence.

File: script/test_in_openshift/README.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+- [run.sh](/script/test_in_openshift/run.sh) is the master script
+  which executes all the tests
+  
+- [lib.sh](/script/test_in_openshift/lib.sh) consists of helper functions
+  for `kompose up` and `kompose down` checks
+  
+- [tests/](/script/test_in_openshift/tests) directory contains the test scripts
+
+- [compose-files/](/script/test_in_openshift/compose-files/) directory contains the docker compose
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: missing period at end of sentence.

File: script/test_in_openshift/README.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+                ├── entrypoint-command.sh
+                ├── etherpad.sh
+                └── redis-replica-2.sh
+                └── ..
+```
+
+- [run.sh](/script/test_in_openshift/run.sh) is the master script
+  which executes all the tests
+  
+- [lib.sh](/script/test_in_openshift/lib.sh) consists of helper functions
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: missing period at end of sentence.

File: script/test_in_openshift/README.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+        ├── run.sh
+        └── tests
+                ├── buildconfig.sh
+                ├── entrypoint-command.sh
+                ├── etherpad.sh
+                └── redis-replica-2.sh
+                └── ..
+```
+
+- [run.sh](/script/test_in_openshift/run.sh) is the master script
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: missing period at end of sentence.

File: script/test_in_openshift/README.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
+# Functional tests for Kompose on OpenShift
+
+## Introduction
+
+The functional tests for Kompose on OpenShift cluster leverages  `oc cluster up` to bring a single-cluster OpenShift instance. The test scripts
+are hosted under script/test_in_openshift. The directory structure is as below:
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: remove random newline. Or do newline after the `script/test_in_openshift` sentence.

File: script/test_in_openshift/README.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+- Docker Compose examples are available under [examples/](/examples] or [/script/test/fixtures](/script/test/fixtures).
+
+## Running OpenShift tests
+
+### Install `oc	cluster	up`
+
+The scripts use [`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) for setting up a single-machine OpenShift cluster. Installing oc binary is a pre-requisite for running these tests
+
+Run `make test-openshift` to run the OpenShift tests
```
Comment (cdrage) [inline script/test_in_openshift/README.md]: Use `make test-openshift` to run the OpenShift tests.

File: script/test_in_openshift/README.md (hunk 11 above)
Comment (cdrage) [reply script/test_in_openshift/README.md]: fix this @ashetty1 then all good from here :+1: feel free to merge after you've added the period.

**Comment by cdrage** [comment]
LGTM!

**Pull request merged**